MapboxSearch/place-autocomplete/src/main/java/com/mapbox/search/autocomplete/PlaceAutocompleteSuggestion.kt

```diff
@@ -27,7 +27,7 @@ public class PlaceAutocompleteSuggestion internal constructor(
     /**
      * Place geographic point.
      */
-    public val coordinate: Point,
+    public val coordinate: Point?,
```

Comment:
If customers don't use it, I'd consider removing this field bacause it's available only in rare cases and may cause misunderstanding.

cc @mapbox-danny

Reply:
The following cases exist when a coordinate may be available in the suggestion:
1) History or Favorites
2) Curb (special backend processing)
3) In the future if we enable the /forward endpoint as an engine option

We should think through if any of these are still needed or desirable.

Reply:
Todo: Find out if Curb is using the coordinate in suggest

Reply:
I kept this field for now according to the feedback above.